appserver/website-bootstrap/network.html

```diff
@@ -46,6 +47,156 @@ <h3>Analyze Network</h3>
             <div class="col-12 col-md-10">
                 <header class="page-header header container-fluid">
                     <svg id="network"></svg>
+					<script>
+
+						var width = 900, height = 600;
+
+						var svg = d3.select("#network")
+							.attr("width", width)
+							.attr("height", height);
+
+						var simulation = d3.forceSimulation()
+							.force("link", d3.forceLink().id(function(d) { return d.id; }))
+							.force("charge", d3.forceManyBody()
+								.strength(-300)
+								.theta(0.8)
+								.distanceMax(150)
+							)
+							.force("center", d3.forceCenter(width / 2, height / 2));
+
+						const graph = {
```

Comment:
this variable name is shadowed by the parameter name inside the 'run(graph)' function. To reduce the risk of bugs, would you mind giving this variable a more specific name to differentiate it from the function parameter?

Reply:
ah actually the parameter name is that graph object. I did rename it to graphData to make it more specific, but they're the same